docs/4.2. Behaviour - Querying.md

```diff
@@ -10,25 +10,25 @@ Where the behaviour associated with an API attribute is not sufficiently clear f
 
 ### Subscriptions
 
-`secure` Indicates whether the WebSocket connection is provided via an encrypted connection (ws:// vs. wss://). Unless otherwise indicated the value of this attribute SHOULD be 'false' if the API is being presented via HTTP, and 'true' for HTTPS. Query API clients MAY choose to specify the opposite value in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
+`secure` Indicates whether the WebSocket connection is provided via an encrypted connection (`ws://` vs. `wss://`). Unless otherwise indicated the value of this attribute SHOULD be `false` if the API is being presented via HTTP, and `true` for HTTPS. Query API clients MAY choose to specify the opposite value in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
 
 `authorization` Indicates whether the WebSocket connection requires authorization in order to connect. Use of authorization is likely to be a deployment decision and be the same for all subscriptions exposed from a single Query API, and for the RESTful API itself. Query API clients MAY specify a requested value for this attribute, but in most circumstances this will result in a 400 response code if the Query API is operating in the opposite mode.
```

Comment:
```suggestion
`authorization`: Indicates whether the WebSocket connection requires authorization in order to connect. Use of authorization is likely to be a deployment decision and be the same for all subscriptions exposed from a single Query API, and for the RESTful API itself. Query API clients MAY specify a requested value for this attribute, but in most circumstances this will result in a 400 response code if the Query API is operating in the opposite mode.
```